libs/vkd3d/resource.c

```diff
@@ -5076,6 +5078,9 @@ static HRESULT d3d12_descriptor_heap_create_descriptor_set(struct d3d12_descript
     VkDescriptorSetAllocateInfo vk_set_info;
     VkResult vr;
 
+    if (vkd3d_descriptor_debug_active_qa_checks() && descriptor_heap->desc.Type == D3D12_DESCRIPTOR_HEAP_TYPE_CBV_SRV_UAV)
```

Comment:
Need account for this in the pool also otherwise it fails to allocate on AMD.

Reply:
Fixed.